### PR TITLE
MCSS-1: Update MN 3D Model

### LINKED ISSUE #1: Create MN 3D Model
> We want to integrate a 3D Model of MN on the frontpage of DeerHacks

# Acceptance Criteria

- [x] Create to-scale blender model of MN building with `glowing transparency` style

### PR DESCRIPTION
**Pull request by hana-dowe**
# What's Inside

- [x] Update MN model

... full details of acceptance criteria documented in the linked GitHub issue

[//]: <> 'Self Checklist When Opening a Pull Request'

[//]: # "Pull Request Title Follows MCSS-<github issue number>: <Issue Title>, e.g. MCSS-123: Example Title (Commit messages do not need to follow convention but recommended)"
[//]: # "All ticket requirements are met. If not, then create a draft PR"
[//]: # "GitHub Issue Pre-Linked to Branch"
[//]: # "Correct Label to this Branch (Can be done after PR is created)"

### CODE REVIEW

**Review by anthonytedja — COMMENTED** [review]
I like the previous darker blue color & opacity :)

File: components/MNModel/index.tsx [hunk 1]
```diff
@@ -31,28 +31,19 @@ const Model = (props: Props) => {
 
   return (
     <group dispose={null}>
-      <mesh
```
Comment (anthonytedja) [inline components/MNModel/index.tsx:34]: I think Cube and Cube_1 were the way to go

Comment (hana-dowe) [reply components/MNModel/index.tsx:34]: bruh

**Review by anthonytedja — APPROVED** [review]
lgtm

**Pull request merged**